solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/ModelQuery.java

```diff
@@ -234,16 +362,16 @@ public Explanation explain(LeafReaderContext context, int doc)
         throws IOException {
       // FIXME: This explain doens't skip null scorers like the scorer()
       // function
-      final Explanation[] explanations = new Explanation[allFeatureValues.length];
-      int index = 0;
-      for (final FeatureWeight feature : allFeatureWeights) {
-        explanations[index++] = feature.explain(context, doc);
-      }
+     // final Explanation[] explanations = new Explanation[extractedFeatureWeights.length];
+    //  for (final FeatureWeight feature : extractedFeatureWeights) {
```

Comment:
Can be deleted as this code is doing exactly the same thing as the old code. I think we do not need an explanation for all features, only need an explanation for how the document is scored. I might be wrong.